src/Dependencies/Collections/ImmutableSegmentedDictionary`2+Builder+KeyCollection.cs

```diff
@@ -2,6 +2,11 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#if CODE_STYLE
+#pragma warning disable CS8601
+#pragma warning disable CS8618
+#endif
+
```

Comment:
These will be fixed by #51065. All the suppressions in this layer can be removed after that change merges.

Reply:
sounds good

Reply:
removed.